server/models/Request.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  user_id: { type: String, max: 50, required: true },
+  sitter_id: { type: String, max: 50, required: true },
+  pet_ids: { type: [String], max: 50, required: true },
```

Comment:
Since these fields refer to another table/collection, it would be best to make them `ref`s instead (you can find more info [here](https://mongoosejs.com/docs/populate.html#populate)).

Pets are a nice addition 🎉

Reply:
Forgot about that, thanks

Reply:
@MilesDev91 I think we call it `petIds` - at a later date we can add in other information like breed, size, age etc to the Pet model  😄